segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerRecoveryUtils.java

```diff
@@ -441,4 +443,106 @@ public static void updateCoreAttributes(Map<Integer, String> backUpMetadataSegme
         Futures.allOf(futures).get(timeout.toMillis(), TimeUnit.MILLISECONDS);
         return backUpMetadataSegments;
     }
+
+    public static boolean matchAttributes(Map<Integer, String> backUpMetadataSegments,
```

Comment:
@andreipaduroiu @sachin-j-joshi  Should I remove this? This method was included for testing purpose to verify if the attributes for each of the segment is correctly updated or not?

Reply:
Actually, I removed it. Please let me know if it's required.